pyproject.toml

```diff
@@ -0,0 +1,20 @@
+[tool.poetry]
+name = "bot"
+version = "0.1.0"
+description = "The TagScript Bot"
+authors = ["Lunarmagpie <Bambolambo0@gmail.com>"]
```

Comment:
Should include me and trag1c here too, since we're all contributors.

Mine is `CircuitSacul <circuitsacul@icloud.com>`, ask @trag1c for his.

Reply:
i was thinking you guys could just PR yourself in so i didn't need to ask for emails :sweat_smile:

Reply:
That works for me